awesome.js

```diff
@@ -336,7 +515,13 @@ class Awesome{
         }
 
         /**
-         * requireCSS requires a CSS stylesheet into the document
+         * requireCSS requires and appends scripts to CSS head
+         *
+         * @example
+         *
+         * //require your component CSS
+         * awesome.requireCSS(`${awesome.path}components/your-component/your-component.css`);
+         *
```

Comment:
should say any or some css, not explicitly your components css. that makes it seem like it is only meant to include your components css, not any css.